example/tools/run-openfoam.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Does the `run-openfoam.sh` we have in the tutorials repository work as well here? Ideally, we will only use that when we move the case to the tutorials, but we can always extend it, if needed.